app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class UploadBddInstructions < Job
+      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_bdd_instructions'
+
+      # retry for one day
+      sidekiq_options retry: 14
+
+      # Submits a BDD instruction PDF in to EVSS
+      #
+      # @param submission_id [Integer] The {Form526Submission} id
+      #
+      def perform(submission_id)
+        super(submission_id)
+        with_tracking('Form526 Upload BDD instructions:', submission.saved_claim_id, submission.id) do
+          file_body = File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
```

Comment:
I'm a fan of [this style](https://github.com/department-of-veterans-affairs/vets-api/blob/8546477896cc1247aa5d2d54866007df13eb5bec/spec/factories/va0994.rb#L6
) of reading a file, but I see this relative, forward-slash-hard-coded style all over vets-api --so it doesn't look like there's a consensus